.github/CODEOWNERS

```diff
@@ -0,0 +1,32 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence, all below
+# owners will be requested for review when someone opens a pull request.
+
+*    @cshilwant @StaticRocket @praneethbajjuri @uditkumarti
+
+# Order is important; the last matching pattern takes the most
+# precedence.
+
+# In this case, below team owns any files in all the below
+# mentioned directories in the root of our repository
+# and any of its subdirectories.
+
+/source/buildroot/ @TexasInstruments/sitara-mpu-linux-sdk
+/source/debian/ @TexasInstruments/sitara-mpu-linux-sdk
+/source/system/ @TexasInstruments/sitara-mpu-linux-sdk
+/source/linux/Overview/ @TexasInstruments/sitara-mpu-linux-sdk
+
+# In this example, below owners own any file in the
+# `/source/linux/Foundational_Components`directory in
+# the root of your repository except for the other mentioned
+# subdirectories, as these subdirectories has its own set of owners
+
+/source/linux/Foundational_Components/ @praneethbajjuri @uditkumarti @r-vignesh
+
+/source/linux/Foundational_Components/Power_Management/ @praneethbajjuri @DhruvaG2000
```

Comment:
LGTM, Would be better in future to add email IDs I think, but for now this is fine.